### DISCUSSION TITLE
Show Media Buttons on Card at all times

### DISCUSSION THREAD

**Discussion by basesnow** [home-assistant/frontend#11243] [Other feature requests]
## The request

The new media card will automatically hides the media buttons (play/pause, forward, and back) as well as the progress bar on my Fire HD 8, since the card size is smaller. This has reduced the functionality of the card, since I now have tap the ellipsis and pull up the pop up screen in order to get at those buttons. 

Is it possible to add the ability to manually turn those buttons and progress bar on. That way, users can decide the functionality of the card, regardless of the card size.

## The alternatives

I have not looked into alternatives

## Additional information

reference issue #5286 
Media card on my fire tablet:
![FireTablet](https://user-images.githubusercontent.com/36908616/77161734-c152ab00-6a80-11ea-8fee-53e61a6d9c6c.jpeg)

Media card on my Mac:
<img width="1183" alt="mac" src="https://user-images.githubusercontent.com/36908616/77161749-cadc1300-6a80-11ea-9627-907db92bae36.png">

**Comment by matthiasdebaat** [comment]
I like this proposal! let's do that.
https://github.com/home-assistant/frontend/discussions/7341#discussion-24683
